pkgs/tools/admin/docker-credential-helpers/default.nix

```diff
@@ -45,5 +45,7 @@ buildGoPackage rec {
     license = licenses.mit;
     maintainers = [ maintainers.marsam ];
     platforms = platforms.linux ++ platforms.darwin;
+  } // lib.optionalAttrs stdenv.isDarwin {
+    mainProgram = "docker-credential-osxkeychain";
```

Comment:
Could you move this to a separate PR? I think having platform-dependent `mainProgram`s warrants some debate.

Reply:
Will do.

Reply:
#171749